src/index.html

```diff
@@ -1,14 +1,52 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" 
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&amp;display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="/moyo.png" 
+        class="imglogo"
+        alt="logo">
+      </a>
```

Comment:
1. you should add classes due to bem, so here if you want to have a separate block `logo` so your image should have class `logo__img`, but to my mind there is no need to create a separate class, it could be an element of header block ->  `header__logo`
2. something is wrong with formatting, every attribute should be from new line  
```suggestion
      <a href="#" class="logo">
        <img 
          src="/moyo.png" 
          class="imglogo"
          alt="logo"
        >
      </a>
```